app/src/app/components/specmath-modal/modal.component.spec.ts

```diff
@@ -47,7 +49,7 @@ describe('ModalComponent', () => {
         MatTooltipModule,
```

Comment:
Since you switched to a stub, you might check whether any of these imports can be removed.  Though I don't see anything obvious.

Reply:
Removed the form modules since they're not being used